### PR TITLE
Basic BRD stuff

### PR DESCRIPTION
**Pull request by Vayhyr**
Still need some work (weave and dots)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,5 +23,9 @@ test.html
 
 # shh
 *.ignore.*
+
+\.vs/
+# Forcing Unix line break
+*.js text eol=lf
```
Comment (NamelessFaceless) [inline .gitignore]: ESLint should be screaming at you for not using LF so I feel like this is unnecessary.

File: src/parser/jobs/brd/songs.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export default class Gauge extends Module {
+	static dependencies = [
+		'cooldowns',
+		'suggestions',
+	]
+
+	_WANDERERS_MINUET = 0
+	_MAGES_BALLAD = 0
+	_ARMYS_PAEON = 0
```
Comment (NamelessFaceless) [inline src/parser/jobs/brd/songs.js]: Should have them like `_wanderersMinuet`, `_magesBallad`, `_armysPaeon`.

**Comment by mousseng** [comment]
any consideration for other low-hanging fruit like Straight Shot uptime, Sidewindering enemies without both dots ([see Summoner](https://github.com/xivanalysis/xivanalysis/blob/fc9195eb3e2b5edebe6dbb79d76b653eb336e4d4/src/parser/jobs/smn/Aetherflow.js#L11)), using Barrage on the correct abilities, etc?

File: src/parser/jobs/brd/about.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import CONTRIBUTORS from 'data/CONTRIBUTORS'
+import CoreAbout from 'parser/core/modules/About'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
+		<Message warning icon>
+			<Icon name="warning sign"/>
+			<Message.Content>
+				Still in development!
```
Comment (ackwell) [inline src/parser/jobs/brd/about.js]: 👍

File: src/data/ZONES.js [hunk 4]
```diff
@@ -97,6 +97,13 @@ const ZONES = {
 		dbId: 30057,
 		banner: 'https://secure.xivdb.com/img/game/112000/112261.png',
 	},
+
+	// UwU
+	UWU:{
```
Comment (ackwell) [inline src/data/ZONES.js]: Ah, ta for that.

File: src/parser/jobs/brd/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import About from './about'
```
Comment (ackwell) [inline src/parser/jobs/brd/index.js]: Just for convention purposes, `PascalCase` for the module files pl0x

File: src/parser/jobs/brd/songs.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
+//import React, {Fragment} from 'react'
+
+//import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+//import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+//import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+export default class Gauge extends Module {
```
Comment (ackwell) [inline src/parser/jobs/brd/songs.js]: \> `songs.js`
\> exports `Gauge`
🤔

File: src/parser/jobs/brd/songs.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+		if (abilityId === ACTIONS.MAGES_BALLAD.id) {
+			this._magesBallad ++
+		}
+		if (abilityId === ACTIONS.ARMYS_PAEON.id) {
+			this._armysPaeon ++
+		}
+	}
+	_onDeath() {
+		// No song up when you lick the floor
+		console.log('Stop licking the floor, thk')
```
Comment (ackwell) [inline src/parser/jobs/brd/songs.js]: Gunna request impl for these two before merge please~

**Comment by ackwell** [comment]
Did you forget to add the other files? It looks like they've all been deletemundo'd.

**Comment by Vayhyr** [comment]
Yeah, they got untracked

**Comment by ackwell** [comment]
This will be merged as part of #50, closing this PR~

**Pull request closed without merging**